### PR TITLE
Delay AssignGear faces+goggles

### PR DESCRIPTION
**Pull request by beaar**
Loadout faces and goggles should now apply after player profile

### CODE REVIEW

**Comment by Freddo3000** [comment]
Should probably merge #300 first

**Comment by beaar** [comment]
> Maybe add a isPlayer check?

For not using waitUntil on AI?

**Comment by Freddo3000** [comment]
> 
> 
> > Maybe add a isPlayer check?
> 
> For not using waitUntil on AI?

Yeah. You could also use `CBA_fnc_waitUntilAndExecute` instead of `CBA_fnc_waitAndExecute`, with the condition evaluated being `{!isPlayer}` and timeout of 1. Should execute immediately, otherwise wait for timeout.

**Comment by beaar** [comment]
Like so?

```
condition:
    !isPlayer _unit
success (AI, now):
    set face/goggles
timeout (player, after 1s):
    set face/goggles
```

I could move it to a setGoggles function (like with setFace) and do an isPlayer skip check in assignGear.sqf, but waitUntilAndExecute with a timeout seems like overengineering the same thing

**Comment by Freddo3000** [comment]
Yeah moving it to a separate function would probably be best

File: addons/assigngear/functions/fn_loadAssignGear.sqf [hunk 1]
```diff
@@ -49,7 +49,13 @@ private _loadoutArray = [];
         CASE("vest",2);
         CASE("backpack",3);
         CASE("headgear",4);
-        CASE("goggles",5);
+        case "goggles": {
```
Comment (Freddo3000) [inline addons/assigngear/functions/fn_loadAssignGear.sqf:52]: lowercase

**Comment by Freddo3000** [comment]
Something that I would suggest adding is a property for loadouts, such as `overwritePlayerIdentity = 0/1`, for setting faces and goggles.

**Comment by beaar** [comment]
> Something that I would suggest adding is a property for loadouts, such as `overwritePlayerIdentity = 0/1`, for setting faces and goggles, though that might be something to do in a separate PR.

I think that's too extra - the expected functionality is you define faces and goggles in a loadout and they're applied to the unit. An additional attribute to opt-in to this feels counter-intuitive.

**Pull request merged**